CloudFormation/Linux/amazon-linux-2023-attestable-image/base-template.yaml

```diff
@@ -0,0 +1,749 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Base infrastructure for building attestable Amazon Linux 2023 AMIs using EC2 Image Builder and kiwi-ng"
+
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: "Resource Configuration"
+        Parameters:
+          - ResourceNamePrefix
+          - ImageBuilderResourceVersion
+      - Label:
+          default: "Infrastructure Configuration"
+        Parameters:
+          - InstanceType
+          - SubnetId
+          - SecurityGroupId
+          - KeyPair
+          - TerminateInstanceOnFailure
+    ParameterLabels:
+      ResourceNamePrefix:
+        default: "Resource Name Prefix"
+      ImageBuilderResourceVersion:
+        default: "Image Builder Resource Version"
+      InstanceType:
+        default: "Instance Type"
+      SubnetId:
+        default: "Subnet ID"
+      SecurityGroupId:
+        default: "Security Group ID"
+      KeyPair:
+        default: "Key Pair"
+      TerminateInstanceOnFailure:
+        default: "Terminate Instance On Failure"
+
+Parameters:
+  ResourceNamePrefix:
+    Type: String
+    Description: Prefix for resource names
+    Default: "attestable-ami"
+  SubnetId:
+    Type: AWS::EC2::Subnet::Id
+    Description: Subnet ID for Image Builder Instance
+  SecurityGroupId:
+    Type: AWS::EC2::SecurityGroup::Id
+    Description: Security Group ID for Image Builder Instance
+  KeyPair:
+    Type: AWS::EC2::KeyPair::KeyName
+    Description: SSH key pair for troubleshooting
+  TerminateInstanceOnFailure:
+    Type: String
+    Description: Terminate Instance On Build Failure
+    Default: true
+    AllowedValues: [true, false]
+  ImageBuilderResourceVersion:
+    Type: String
+    Description: Image Builder Resource Version
+    Default: "1.0.0"
```

Comment:
Not blocking: Given the release of [auto-versioning](https://aws.amazon.com/about-aws/whats-new/2025/11/ec2-image-builder-auto-versioning-infrastructure/) a few days ago, we could make this "1.0.x" and the version would never need manually updating if the resources changes.

If you change this, it'll need to be updated in the README as well where its referenced.